pipenv/utils.py

```diff
@@ -536,7 +569,7 @@ def resolve_deps(deps, which, which_pip, project, sources=None, verbose=False, p
     markers_lookup = {}
 
     python_path = which('python', allow_global=allow_global)
-    backup_python_path = shellquote(sys.executable)
+    backup_python_path = sys.executable
```

Comment:
Was this related to this PR, or something bundled in with merging the other missing commits?

Reply:
This was a PR, #1241 -- since the python path here is never actually used as an argument to a subprocess call, it doesn't make sense to shellquote it (and it was breaking something for someone)